### ISSUE TITLE
Add dependency on C/C++ and Java extensions

### ISSUE THREAD

**Issue by pgfisico** [wpilibsuite/vscode-wpilib#82]
Rather than require users to install the extensions for the complete and best experience, add a dependency on them.

Specifically:
- ms-vscode.cpptools
- vscjava.vscode-java-pack

**Comment by pgfisico** [comment]
Also noticed https://github.com/Microsoft/vscode-cpptools/issues/1670 regarding offline installs.

Support for dependencies from the marketplace was added for VSIX installs (https://github.com/Microsoft/vscode/issues/15477), however I can see the use case for having a fully offline package.

**Comment by ThadHouse** [comment]
Yeah offline is a huge part of our install base, and what is likely going to be the primary distribution method for a majority of teams.

To solve the dep issue we will likely just create an FRC pack. That solves a lot of problems and enables a lot of use cases.
